### ISSUE TITLE
Feature: set command (For changing settings)

### ISSUE THREAD

**Issue by trymeouteh** [Schniz/fnm#1199]
A `set` command to change settings in fnm.

Here is an example on how to modify the settings in fnm...
```
$ fnm set use-on-cd enable
$ fnm set version-file-strategy recursive
```

**Comment by Schniz** [comment]
yeah i thought about having a `.fnmrc` with settings instead of putting it all in `fnm env` but I don't want to have too much of file system dependencies, and it'll make debugging issues harder for others

maybe you can try doing the following experiment:

```bash
echo '--use-on-cd --version-file-strategy=recursive' > ~/.fnmrc # (can contain linebreaks instead of spaces i believe)
```
and then change your shell profile to have
```bash
eval "$(cat ~/.fnmrc | xargs fnm env)"
```

and tell us if it's worth it?
